hack/kind/setup.sh

```diff
@@ -77,11 +77,23 @@ setup_cluster() {
   kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
 }
 
+install_kubectl() {
+    if ! command -v kubectl &> /dev/null
+then
+    info "kubectl not found, attempting to install"
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+    chmod +x kubectl
+    export PATH="$(pwd):$PATH"
+fi
```

Comment:
Unrelated change :( 

Also ... IMHO, we shouldn't install tools / make changes to users system other than in locations known/announced to them. E.g. `make tools` is explicit and installs to `<prj>/tmp/bin`